dagger-core/src/main/java/com/gotocompany/dagger/core/sink/kafka/util/SinkKafkaConfigUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.gotocompany.dagger.core.sink.kafka.util;
+
+import com.gotocompany.dagger.common.configuration.Configuration;
+
+import java.util.Properties;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SinkKafkaConfigUtil {
+    private static final Pattern SINK_KAFKA_BUILT_IN_CONFIG_REGEX = Pattern.compile("SINK_KAFKA_BUILT_IN_CONFIG_(.*)");
```

Comment:
For any built in sink kafka configs we need to have predefined prefix to separate from custom config